src/components/Dialogs/Dialog/Dialog.scss

```diff
@@ -1,21 +1,35 @@
 .Dialog {
     width: 400px;
+    border-radius: 12px;
+
+    &__header {
+        padding: var(--space-200);
+        display: flex;
+        justify-content: space-between;
+        align-items: flex-start;
+    }
 
     &__content {
+        max-height: 528px;
         padding: var(--space-200);
+        overflow-y: auto;
+        white-space: normal;
     }
 
     &__actions {
         display: flex;
-        justify-content: flex-end;
+        justify-content: flex-start;
         align-content: baseline;
         margin-top: var(--space-75);
```

Comment:
not sure about this margin-top. The text is disappearing before the line because of the margin.
<img width="481" alt="image" src="https://github.com/user-attachments/assets/c696c020-eb5b-454a-9ca1-12b1c786cb2a">
 Please check with the designers.